packages/multi-test/src/test_helpers/contracts/echo.rs

```diff
@@ -0,0 +1,62 @@
+//! Very simple echoing contract which just returns incomming string if any, but performming subcall of
+//! given message to test response
+
+use cosmwasm_std::{
+    to_binary, Binary, ContractResult, Deps, DepsMut, Env, MessageInfo, Reply, Response, StdError,
+    SubMsg, SubMsgExecutionResponse,
+};
+use serde::{Deserialize, Serialize};
+
+use crate::{test_helpers::EmptyMsg, Contract, ContractWrapper};
+
+#[derive(Debug, Clone, Serialize, Deserialize, Default)]
+pub struct Message {
+    pub data: Option<String>,
+    pub sub_msg: Vec<SubMsg<Binary>>,
+}
+
+fn instantiate(
+    _deps: DepsMut,
+    _env: Env,
+    _info: MessageInfo,
+    _msg: EmptyMsg,
+) -> Result<Response<Binary>, StdError> {
```

Comment:
Commented above, but I am quite sure there is no need for `Response<Binary>` unless you do something like:

```rust
let msg = CosmosMsg::Custom(Binary::from(b"foo"));
res.add_message(msg);
```

The type here is unused unless you use some blockchain-specific custom return types. Which is only the case for tgrade-specific contracts (PoE), terra-specific contracts, etc. Not for any tests. Multi-test does not support those messages when returned